dependencies.gradle

```diff
@@ -1,71 +1,66 @@
-            glideOkHttp3       : "com.github.bumptech.glide:okhttp3-integration:${glideVersion}",
-            jodaTimeAndroid    : 'net.danlew:android.joda:2.9.9.3',
-            kotlin             : "org.jetbrains.kotlin:kotlin-stdlib-jre7:${kotlinVersion}",
-            kotlinReflect      : "org.jetbrains.kotlin:kotlin-reflect:${kotlinVersion}",
-            moshi              : "com.squareup.moshi:moshi:${moshiVersion}",
-            moshiKotlin        : "com.squareup.moshi:moshi-kotlin:${moshiVersion}",
-            playServicesAuth   : "com.google.android.gms:play-services-auth:${playServicesVersion}",
-            rxAndroid          : 'io.reactivex.rxjava2:rxandroid:2.0.2',
-            rxJava             : 'io.reactivex.rxjava2:rxjava:2.1.12',
-            rxLint             : 'nl.littlerobots.rxlint:rxlint:1.6.1',
-            supportAnnotations : "com.android.support:support-annotations:${supportLibVersion}",
-            supportAppCompat   : "com.android.support:appcompat-v7:${supportLibVersion}",
-            supportConstraint  : 'com.android.support.constraint:constraint-layout:1.1.0',
-            supportDesign      : "com.android.support:design:${supportLibVersion}",
-            timber             : 'com.jakewharton.timber:timber:4.7.0',
-            viewPagerAdapter   : 'com.novoda:view-pager-adapter:1.0.0'
+            algolia           : 'com.algolia:algoliasearch-android:3.23.13',
+            androidKtx        : 'androidx.core:core-ktx:0.2',
+            arrowCore         : 'io.arrow-kt:arrow-core:0.7.2',
+            crashlytics       : 'com.crashlytics.sdk.android:crashlytics:2.9.4',
+            dagger            : "com.google.dagger:dagger:${daggerVersion}",
+            daggerCompiler    : "com.google.dagger:dagger-compiler:${daggerVersion}",
+            firebaseAuth      : 'com.google.firebase:firebase-auth:16.0.2',
+            firebaseConfig    : 'com.google.firebase:firebase-config:16.0.0',
+            firebaseCore      : 'com.google.firebase:firebase-core:16.0.1',
+            firebaseDb        : 'com.google.firebase:firebase-database:16.0.1',
+            firebaseFirestore : 'com.google.firebase:firebase-firestore:17.0.2',
+            firebaseMessaging : 'com.google.firebase:firebase-messaging:17.0.0',
+            firebasePerf      : 'com.google.firebase:firebase-perf:16.0.0',
+            firebaseStorage   : 'com.google.firebase:firebase-storage:16.0.1',
+            firebaseUiStorage : 'com.firebaseui:firebase-ui-storage:4.0.1',
+            flexbox           : 'com.google.android:flexbox:1.0.0',
+            glide             : "com.github.bumptech.glide:glide:${glideVersion}",
+            glideCompiler     : "com.github.bumptech.glide:compiler:${glideVersion}",
+            glideOkHttp3      : "com.github.bumptech.glide:okhttp3-integration:${glideVersion}",
+            jodaTimeAndroid   : 'net.danlew:android.joda:2.9.9.4',
```

Comment:
worth having an issue to use JSR-310/TreeTenABP?

Reply:
Yeah we should migrate eventually. Can you file it please?